test/e2e/operator_groups_e2e_test.go

```diff
@@ -80,13 +80,25 @@ func patchOlmDeployment(t *testing.T, c operatorclient.ClientInterface, newNames
 }
 
 func checkOperatorGroupAnnotations(obj metav1.Object, op *v1alpha2.OperatorGroup, targetNamespaces string) error {
```

Comment:
What about just passing in the boolean `mustHaveTargetNamespaces` as an arg?

Just thinking about when I'm reading back over this in the future and I come across the `"-"` vs looking up the function signature and seeing a boolean with a descriptive name.

Reply:
(or even having two separate functions, on for checking targets and one for not)

Reply:
I saw some precedent of using "-" in k8s, but what you suggested is better.